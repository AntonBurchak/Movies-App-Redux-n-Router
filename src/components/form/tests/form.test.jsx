import React from 'react'; 
import Form from '../form'
import { shallow } from 'enzyme'

describe('<Form />', () => {
    let wrapper;
    let instance;

    const props = {
        text: 'Initial text',
        handleInputChange: jest.fn(),
        handleSubmit: jest.fn()
    }

    beforeEach(() => {
        wrapper = shallow(<Form { ...props }/>);
        instance = wrapper.instance();
    });

    it('should render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('should render h1', () => {
        
        expect(wrapper.find('h1')).toHaveLength(1);
    });
});