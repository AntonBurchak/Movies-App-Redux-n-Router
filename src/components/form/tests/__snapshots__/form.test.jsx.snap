// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should render correctly 1`] = `"<div><h1>Initial text</h1><input type=\\"text\\" id=\\"login\\"/><input type=\\"text\\" id=\\"password\\"/><button class=\\"submit\\">Submit</button></div>"`;
